core/src/lib.rs

```diff
@@ -291,6 +292,45 @@ impl Schema {
     pub fn identifies_attribute(&self, x: &NamespacedKeyword) -> bool {
         self.get_entid(x).map(|e| self.is_attribute(e)).unwrap_or(false)
     }
+
+    pub fn as_edn_value(&self) -> edn::Value {
+        let mut all_values = Vec::new();
+        for (entid, attribute) in &self.schema_map {
+            let mut attribute_map: BTreeMap<edn::Value, edn::Value> = BTreeMap::default();
+            attribute_map.insert(edn::Value::NamespacedKeyword(NamespacedKeyword::new("db", "id")), edn::Value::Keyword(Keyword::new(entid.to_string())));
```

Comment:
So, there are no values defined for `index`, `component` and `fulltext`, but they are present as entids in `entids.rs`.

```

pub const DB_IS_COMPONENT: Entid = 10;
pub const DB_INDEX: Entid = 11;
pub const DB_FULLTEXT: Entid = 12;
```

I am therefore going to create entries for them in `values` as clearly they exist as concepts here. If you feel that is not correct, let me know.

There is no entry anywhere for `db/id` so I shall remove that entry from the output.